test/loopback-dl.test.js

```diff
@@ -1473,6 +1483,20 @@ describe('DataAccessObject', function() {
     assert(error, 'An error should have been thrown');
   });
 
+  it('should throw when malformed logical operators follow valid logical clauses', function() {
```

Comment:
@superkhau,

I would personal say we should not care about whether test starts with `should` or not, however I have seen you are so careful about that in PRs and have documented [here](https://github.com/strongloop/loopback.io/blob/gh-pages/pages/en/contrib/style-guide.md#test-naming), so I followed that. If it's me I would say most of the tests currently start with `should` and we should follow that or at least we should follow what the majority of tests use in that specific file and follow them to have the tests in that file consistent.

I also propose to delete the following like from https://github.com/strongloop/loopback.io/blob/gh-pages/pages/en/contrib/style-guide.md#test-naming:

```
Use imperative mood, do not start test names with should.
```